packages/cw-croncat-core/examples/schema.rs

```diff
@@ -40,6 +40,11 @@ fn main() {
         &out_dir,
         "GetTaskResponse",
     );
+    export_schema_with_title(
+        &schema_for!(Option<TaskWithRulesResponse>),
+        &out_dir,
+        "GetTasksWithRulesResponse",
+    );
```

Comment:
Just checked `GetTaskWithRules` method

```suggestion
    export_schema_with_title(
        &schema_for!(Vec<TaskWithRulesResponse>),
        &out_dir,
        "GetTasksWithRulesResponse",
    );
```